project/plugins.sbt

```diff
@@ -3,15 +3,20 @@ resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts
   Resolver.ivyStylePatterns
 )
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.8.0")
+// To resolve a bug with version 2.x.x of the scoverage plugin - https://github.com/sbt/sbt/issues/6997
+// Try to remove when sbt 1.8.0+ and scoverage is 2.0.7+
+ThisBuild / libraryDependencySchemes += "org.scala-lang.modules" %% "scala-xml" % VersionScheme.Always
 
-addSbtPlugin("org.scoverage" %% "sbt-scoverage" % "2.0.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.9.0")
 
-addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0")
+addSbtPlugin("org.scoverage" %% "sbt-scoverage" % "2.0.7")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.1.0")
+addSbtPlugin("com.beautiful-scala" % "sbt-scalastyle" % "1.5.1")
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.16")
+addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.2.0")
 
-addSbtPlugin("com.timushev.sbt" % "sbt-updates"  % "0.6.3")
-addSbtPlugin("org.scalameta"    % "sbt-scalafmt" % "2.4.6")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.19")
+
```

Comment:
could you remove all new lines in between addSbtPlugins, as formatter ignore formatting because of that